app/jobs/bag_move_job.rb

```diff
@@ -54,7 +54,35 @@ def record_failure
     end
   end
 
-  def externally_validates?
+  def bag_includes_metadata?
```

Comment:
The invocation of these validations doesn't seem to belong here, but I recognize that a single-job approach is desirable right now.  However, the knowledge of how to carry out those validations should likely live elsewhere, such as on a validator class that cooperates with a ChipmunkBag instance.